app/src/main/res/layout/fragment_all_list.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
NIT: do you think it would be possible to group together then name and dropdown menu icon together in some sort of a box/outline/something so that it is more distinctly a group? something like, maybe?
![image](https://github.com/user-attachments/assets/d8328966-2456-4dcb-8e3f-802a9de9ed43)

Reply:
How is this?
![image](https://github.com/user-attachments/assets/3a6b6353-5b45-42f3-91af-2e53b2c99125)

Reply:
like a tiny bit lighter imo 🙏